### DISCUSSION TITLE
Acerca de duda con la practica del pokedex, en el const url, entiendo que la parte ultima se marca en azul para poder leer encontrar tal palabra, pero no se por que sigue asi, y me marca error al querer buscar los pokemones. , se debe de instalar algo para que fucnione?

### DISCUSSION THREAD

**Discussion by MarceloCortesC** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#438] [Q&A]
![1s](https://user-images.githubusercontent.com/99509464/158743273-834b54f8-0dc2-4f22-9eff-bd7182edc5c2.JPG)

**Comment by GabrielVillagran** [comment]
hola que tal, cambia  ' por comillas invertidas, son las siguientes: `
![image](https://user-images.githubusercontent.com/44887537/158747641-7af336c5-8ad2-4351-9a71-731b676605e8.png)

**Comment by micheldavrdz** [comment]
Para expandir un poco mas la respuesta de @GabrielVillagran, las grave accents o backticks ( `` ) se añadieron a javascript con ES2015 para agregar lo que conocemos como _template literals_ (plantillas literales o de cadena) y sirven para hacer lo mismo que haciamos en Python usando f-strings al agregar una variable ( _${una_expresion}_ ) a una cadena de texto, poder utilizar comillas simples y dobles. entre otras cosas.
